src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    class="form"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text"
+          name="Surname"
+          autocomplete="off"
+          >
```

Comment:
Didn't fixed. Fix indent in all places
```suggestion
          <input type="text"
            name="Surname"
            autocomplete="off"
          >
```

Reply:
@IvanFesenko, now I got it, should be good to go.